swatch-core/src/main/java/org/candlepin/subscriptions/db/EventRecordRepository.java

```diff
@@ -73,10 +74,10 @@ Stream<EventRecord> findByOrgIdAndTimestampGreaterThanEqualAndTimestampLessThanO
    * @return Stream of EventRecords
    */
   Stream<EventRecord>
-      findByOrgIdAndEventSourceAndEventTypeAndTimestampGreaterThanEqualAndTimestampLessThanOrderByTimestamp(
+      findByOrgIdAndEventSourceAndEventTypeIsInAndTimestampGreaterThanEqualAndTimestampLessThanOrderByTimestamp(
```

Comment:
Can we alias this method?  That name is just too long to be able to easily comprehend.  We can just do something like 

```java
interface UserRepository extends JpaRepository<User, Long> {
  // don't use that crazy long method! use getByEmail instead
  User findFirstByEmailContainsIgnoreCaseAndField1NotNullAndField2NotNullAndField3NotNullAndField4NotNullAndField5NotNullAndField6NotNull(final String email);

  default User getByEmail(final String email) {
    return findFirstByEmailContainsIgnoreCaseAndField1NotNullAndField2NotNullAndField3NotNullAndField4NotNullAndField5NotNullAndField6NotNull(email);
  }
}
```


(from https://stackoverflow.com/a/53315665/6124862)

Reply:
I have wrapped the code as shown in example.